assets/src/edit-story/components/inspector/design/designInspector.js

```diff
@@ -26,21 +26,23 @@ import { LayerPanel } from '../../panels';
 import DesignPanels from './designPanels';
 
 const Wrapper = styled.div`
-  height: 100%;
+  height: calc(100% - 40px);
```

Comment:
What is `40px`? Where does it come from? Is there a way to track it here as a const from another browser?

Reply:
It's necessary because of [this](https://github.com/google/web-stories-wp/blob/master/assets/src/edit-story/components/inspector/inspectorLayout.js#L34) tabs declaration. I can share a constant between both. WDYT?

Reply:
Constant sounds good, but we should ideally just rework the tabs there. Can probably implement those without grid. See #933.

Reply:
Interesting @swissspidy, let me take a look at it.  Thanks for bringing this here.